lib/cell.rb

```diff
@@ -37,7 +37,7 @@ def fire_upon
   end
 
   def render(show_ship = false)
-    if @ship && @fired_upon
+    if @ship && @fired_upon && !@ship.sunk?
```

Comment:
@amymoller realized we needed to add this line to the first conditional on cell.render after playing around with some testing.